boards/arm/atb1103_golden/Kconfig.board

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2019 Actions (Zhuhai) Technology Co., Ltd
```

Comment:
Why copyright to Actions?

Reply:
Ah, see you commented in the PR on this.  Is there a reference to their original code?

Reply:
I have inherited code from them

Reply:
> Ah, see you commented in the PR on this. Is there a reference to their original code?

No. They just shared a zip file with me based on LTS version and I had to clean a lot for upstream.